requirements.txt

```diff
@@ -1,4 +1,5 @@
-retworkx>=0.10.1
+contextvars>=2.4;python_version<'3.7'
+retworkx @ git+https://github.com/georgios-ts/retworkx@pr-dijkstra-visitor
```

Comment:
I think something went funny here adding `contextvars` up top?  I don't see it as used, and it's previously just been at the bottom of the file.

Reply:
Yeah, apparently I messed this up while updating from main. Fixed in 820aab6.